generator/integration_tests/tests/golden_kitchen_sink_metadata_decorator_test.cc

```diff
@@ -322,7 +322,7 @@ TEST_F(MetadataDecoratorTest, ExplicitRouting) {
   // `google.api.routing` for Example 9:
   //
   // https://github.com/googleapis/googleapis/blob/70147caca58ebf4c8cd7b96f5d569a72723e11c1/google/api/routing.proto#L387-L390
-  std::string expected1 = "table_location=instances/instance_bar";
+  std::string expected1 = "table_location=instances%2Finstance_bar";
```

Comment:
Hm, so I didn't consider that. Now that I am considering it, I think I ever so slightly prefer the code the way it is.

We do not URL encode the `=`, so it would look like:
```cc
auto expected1 = "table_location=" + internal::UrlEncode("instances/instance_bar");
```